tests/rados/test_bluestore_features.py

```diff
@@ -95,6 +100,48 @@ def run(ceph_cluster, **kw):
         log.info(f"The relative memory growth for the osd.{osd_id} is {mem_growth} ")
 
     rados_object.change_heap_profiler_state(osd_list, "stop")
+    execution_time = config["execution-time"]
+    log.info(f"Checking osd memery growth for {execution_time} minutes")
+
+    # Setting the scheduled scrub parameters to enable scrubbing continiously
+    if scrub_object.get_osd_configuration("osd_scrub_begin_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_hour", 0)
+        log.info("The osd_scrub_begin_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_hour", 0)
+        log.info("The osd_scrub_end_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_begin_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_week_day", 0)
+        log.info("The osd_scrub_begin_week_day is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_week_day", 0)
+        log.info("The osd_scrub_end_week_day is set to 0")
+
+    # Stroring the initial memory
+    for osd in osd_list:
+        memory = get_mempool_bytes(rados_object, osd)
+        osd_mem_pool[osd] = memory
+
+    # Intentional sleep
+    time.sleep(int(execution_time) * 60)
```

Comment:
Let's reassess the sleep of 30 minutes in the code in the Next PR. please add a backlog task, so that we can add a smart sleep instead of blind sleep?